utilities.js

```diff
@@ -122,7 +122,7 @@ export async function extractNpmArchiveTo(packageBuffer, target) {
 
 export async function trackProgress(cb) {
 
-    let pace = new Progress(`:bar :current/:total (:elapseds)`, {width: 80, total: 1});
+    let pace = new Progress(`:bar :current/:total (:elapsed)`, {width: 80, total: 1});
```

Comment:
It's possible that `:elapseds` is a real thing, if so, I'll happily file a bug against whichever group is using it, because that's awful.